target/build-docker.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I think we want to keep this as part of the main Makefile versus a separate build script. Any reason that won't work?

Reply:
Part of unit testing that was left behind in the commit because I'm a sloppy dev :)